### PR TITLE
fix(PN-13212)  chip stato notifica/pagamenti

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description
delete default role in chip for status of notification

## List of changes proposed in this pull request
- insert role cell for StatusTooltip component

## How to test
in table of notification, with voice over activeted , voice over read status of notification and description explained in the tooltip, only description is clickable

### CODE REVIEW

**Review by SarahDonvito — APPROVED** [review]

**Review by SarahDonvito — APPROVED** [review]

File: packages/pn-commons/src/components/Notifications/StatusTooltip.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ const StatusTooltip = ({
         id={`status-chip-${label}`}
         label={label}
         color={color}
+        role='cell'
```
Comment (AndreaCimini90) [inline packages/pn-commons/src/components/Notifications/StatusTooltip.tsx:29]: This is not completely correct. The problem is into the CustomTooltip component. There we do this

```
{cloneElement(children, {
            onClick: handleTooltipOpen,
          })}
```
In this way we are cloning the children component and we are setting the onClick props. In our case the children component is the Chip component and a Chip component with onClick props behaves as a button (https://mui.com/material-ui/react-chip/#chip-actions).
To avoid this we have to avoid the onClick props if openOnClick props is undefined.

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-commons/src/components/CustomTooltip.tsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ const CustomTooltip: React.FC<Props> = ({
           {...tooltipProps}
         >
           {cloneElement(children, {
```
Comment (AndreaCimini90) [inline packages/pn-commons/src/components/CustomTooltip.tsx:55]: cloneElement is an expensive operation. we can avoid this if openOnClick is false

```
{openOnClick ? cloneElement(children, {
            onClick: handleTooltipOpen,
          }) : children}
```

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-commons/src/components/__test__/CustomTooltip.test.tsx [hunk 3]
```diff
@@ -11,7 +11,7 @@ describe('CustomTooltip Component', () => {
     await act(async () => {
       result = render(
         <CustomTooltip tooltipContent="Mocked content" openOnClick={false}>
-          <p>Mocked Text</p>
+          <button onClick={false ? ()=>{}: undefined}>Mocked Text</button>
```
Comment (AndreaCimini90) [inline packages/pn-commons/src/components/__test__/CustomTooltip.test.tsx:14]: Why this change?

File: packages/pn-commons/src/components/__test__/CustomTooltip.test.tsx [hunk 4]
```diff
@@ -28,7 +28,7 @@ describe('CustomTooltip Component', () => {
           openOnClick={false}
           onOpen={mockOnOpenCallback}
         >
-          <p data-testid="testTooltipText">Mocked Text</p>
+          <button data-testid="testTooltipText" onClick={false ? ()=>{}: undefined}>Mocked Text</button>
```
Comment (AndreaCimini90) [inline packages/pn-commons/src/components/__test__/CustomTooltip.test.tsx:31]: Why this change?

File: packages/pn-commons/src/components/__test__/CustomTooltip.test.tsx [hunk 5]
```diff
@@ -54,10 +54,10 @@ describe('CustomTooltip Component', () => {
     // render component
     const { container, getByRole } = render(
       <CustomTooltip tooltipContent="Mocked content" openOnClick={true}>
-        <p>Mocked Text</p>
-      </CustomTooltip>
+        <button onClick={true ? ()=>{}: undefined}>Mocked Text</button>
```
Comment (AndreaCimini90) [inline packages/pn-commons/src/components/__test__/CustomTooltip.test.tsx:57]: Why this change?

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-commons/src/components/__test__/CustomTooltip.test.tsx (hunk 5 above)
Comment (leleOFA) [reply packages/pn-commons/src/components/__test__/CustomTooltip.test.tsx:57]: by entering the condition I had to insert a react element or an interactable element to insert the onclick

File: packages/pn-commons/src/components/CustomTooltip.tsx (hunk 2 above)
Comment (leleOFA) [reply packages/pn-commons/src/components/CustomTooltip.tsx:55]: done
